text/0041-extensible-block-body.md

```diff
@@ -176,20 +192,25 @@ where
 Hashing an option is similar.  Once again the sentinel hash `hash(&[])` is used to denote absence.
```

Comment:
I think the `hash(&[])` here should be changed to `SENTINEL0`, shouldn't it? And it won't be "once again" anymore, then :wink: